src/lib/cli-util.js

```diff
@@ -0,0 +1,41 @@
+var size = require('window-size')
+var availableWidth = size.width || /*istanbul ignore next */ 80
```

Comment:
I like these sort of `/*istanbul ignore next */`, I personally don't write test for these default conditions 👍